src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -84,8 +84,8 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
                         },
                         {root: true});
                 }
-
                 commit(ModelCalibrateMutation.Calibrated);
+                commit({type: ModelCalibrateMutation.UploadMetadata, payload: data.uploadMetadata});
```

Comment:
Rather than adding an extra mutation that is only ever used here, I wonder if this should either just be part of `ModelCalibrateMutation.Calibrated` (if the uploadMetadata is genuinely integral to the calibration result) or an ADR mutation (if it's only related to uploads).